cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,41 @@
+import time
+from os import getenv
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        db_conn = self.establish_db_connection()
+        if db_conn:
+            self.stdout.write(self.style.SUCCESS(
+                "Database connection established!"
+            ))
+        else:
+            self.stdout.write(self.style.ERROR(
+                "Database not available after max attempts."
+            ))
+
+    def establish_db_connection(self):
+        max_attempts = int(getenv("MAX_ATTEMPTS"))
```

Comment:
The 'MAX_ATTEMPTS' environment variable is being fetched every time the 'establish_db_connection' function is called. This can be inefficient if the function is called multiple times. It would be better to fetch this value once and store it as an instance variable in the 'Command' class.